teknologiradar.csv

```diff
@@ -0,0 +1,16 @@
+Google DataStore,Avvent,Infrastruktur,FALSE,"Skalerbar teknologi for NoSQL databaser<br>Mer info her:<a href=""https://cloud.google.com/datastore"">DataStore</a>"
+Google CloudSQL,Ta i bruk,Infrastruktur,FALSE,"Selvbetejent relasjonsdatabseløsning fra Google som tilbyr MySQL,PostgreSQL og SQL Server <br>Mer info her:<a href=""https://cloud.google.com/sql"">CloudSQL</a>"
+Cloud Composer,Avvent,Infrastruktur,FALSE,"Denne tjenesten fra Google bygger på Apache Airflow for orkestrering av pipeline i Python. Med denne teknologien kan man automatisere produksjonsløp og samtidig få et brukergrensesnitt hvor man kan monitorere kjøringene. <br>Mer info her:<a href=""https://cloud.google.com/composer"">Cloud Composer</a>"
+Google Cloud DLP,Avvent,Infrastruktur,FALSE,"Relativ ny teknologi fra Google for å verne sensitive data. Kan f.eks. brukes til pseudonymisering av data. <br>Mer info her:<a href=""https://cloud.google.com/dlp"">DLP</a>"
+Grafana,Ta i bruk,DevOps,FALSE,"Teknologi for å visulalisere data fra mange ulike kilder. Mest brukt til å visualisere logger og metrikker. Brukerne lage egne fleksible dashboard. <br>Mer info her:<a href=""https://grafana.com"">Grafana</a>"
+Github,Ta i bruk,DevOps,FALSE,"Teknologi for å koordinere prosjekter"
+Terraform,Ta i bruk,Programmering,FALSE,"Infrastruktur som kode som gjør at man kan bygge, endre og versjonere infrastruktur raskt og effektivt <br>Mer info her:<a href=""https://www.terraform.io"">Terraform</a>"
+MOVEit,Ta i bruk,Infrastruktur,FALSE,"Overføring av filer på bakkenivå"
+Apache Spark,Ta i bruk,Programmering,FALSE,"Teknologi for minnebasert distribuert prosessering i klynger eller prosessering på enkel noder. Koden kan skrives i flere språk. Denne teknologien er en del av DAPLA. <br>Mer info her:<a href=""https://spark.apache.org"">Apache Spark</a>"
+NuGet,Ta i bruk,DevOps,TRUE,"Defacto standard-verktøy for pakkehåndtering og kodedeling i .NET som kan brukes i .Net baserte prosjekter både i PxWeb og Altinn. <br>Mer info her:<a href=""https://www.nuget.org"">NuGet</a>"
+Poetry,Prøv ut,DevOps,TRUE,"Brukervennlig Python verktøy for å håndtere avhengigheter og publisere pakker til artifaktlager <br>Mer info her:<a href=""https://python-poetry.org/"">Poetry</a>"
+Google Artifact Registry,Prøv ut,DevOps,TRUE,"Lagring og distribusjon av Python-pakker som er interne i SSB. <br>Generelt om GAR for Python: <a href=""https://cloud.google.com/artifact-registry/docs/python"">GAR Python</a><br>Implementasjonen i skyplattformen: <a href=""https://docs.bip.ssb.no/develop/artifact_registry/#set-up-artifact-registry-for-python"">SSB Developer Guide</a>"
```

Comment:
Denne burde være generelt å "Ta i bruk" ikke bare for python, men for docker osv.